examples/seismic/tutorials/06_bis_elasitc_varying_parameters.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
please find another name for this new tutorial (also, there's a typo)

Reply:
06_elastic_...

Reply:
Why? I like this name it's a tutorial name explicitly says what it is